source/deploy-guide.md

```diff
@@ -13,6 +13,8 @@ This guide will cover:
 - Using the Meteor CLI tool to deploy the application
 - Configuring domains and SSL Encryption for the application
 
+If you desire to deploy your app using your machine, you can continue reading this. But we have an easier and faster way of deploying your app, which is by using the feature [Push To Deploy](/push-to-deploy.html), and if this is your first time deploying a Meteor application, this is what we'd recommend you to use.
```

Comment:
This page should be partially replaced by Push to Deploy.

It makes no sense to have both pages as Push to Deploy is just one way to deploy.

The Push to Deploy page needs to be deleted probably, we could keep the page for now but just with a link to there. (or a redirect, I believe this is possible in hexo config)